iree/tools/CMakeLists.txt

```diff
@@ -34,6 +34,30 @@ if(${IREE_BUILD_COMPILER})
     tensorflow::mlir_xla
   )
 
+  iree_cc_binary(
+    NAME
+      iree_tblgen
+    OUT
+      iree-tblgen
+    SRCS
+      "${IREE_ROOT_DIR}/third_party/mlir/tools/mlir-tblgen/mlir-tblgen.cpp"
+      "${IREE_ROOT_DIR}/iree/compiler/Dialect/VM/Tools/VMOpEncoderGen.cpp"
```

Comment:
Can this depend on iree::compiler::Dialect::VM::Tools instead of pulling in these SRCS and DEPS directly?

It should end up looking close to the Bazel BUILD file: https://github.com/google/iree/blob/baa78198aea49c0fbd0c37ae6fb8514deb9bd37f/iree/tools/BUILD#L105-L116

Reply:
I initially linked against `iree::compiler::Dialect::VM::Tools`, but somehow that wasn't sufficient to enable the necessary command line arguments like `-gen-iree-vm-op-encoder-defs`.

Reply:
Ah, as `iree::compiler::Dialect::VM::Tools` is marked `ALWAYSLINK`, my fix for #190 should work (the additional arguments will appear). If this is needed until my work there is ready and merged, we could merge this as-is and I can make the switch as part of my change.

Reply:
:+1: